scripts/run-integration-tests.sh

```diff
@@ -15,8 +15,8 @@ TESTS_TO_RUN="${TESTS_TO_RUN:-all}"
 
 if [[ "$TESTS_TO_RUN" == "all" ]]; then
   echo "run all tests"
-  pytest -vv -s
+  pytest -vv -s --session-timeout=3600 --timeout=3600
```

Comment:
```suggestion
  pytest -vv -s --session-timeout=3600
```

There seems no point to set the per-test timeout the same as session timeout.

Reply:
seems need both since might stuck on yield [ethermint](https://github.com/crypto-org-chain/ethermint/blob/develop/tests/integration_tests/network.py#L151), can't limit with single session-timeout

Reply:
Won't the session timeout terminate it if it get stuck?

Reply:
no terminate, maybe since test hasn't really begin, it's waiting [ethermint](https://github.com/crypto-org-chain/ethermint/blob/develop/tests/integration_tests/test_priority.py#L123) to be ready